modules/hivemq-edge-module-opcua/src/main/java/com/hivemq/edge/adapters/opcua/OpcUaAdapterConfig.java

```diff
@@ -127,6 +127,10 @@ public void setSecurity(final @NotNull Security security) {
         return overrideUri;
     }
 
+    public void setOverrideUri(final @NotNull Boolean overrideUri) {
+        this.overrideUri = overrideUri;
+    }
+
```

Comment:
```suggestion
```

Please do not add setters to config objects!

Reply:
Added for consistency since everything has a getter/setter in the class.
We decided to add it and have a general session where we clean up all the config objects to rely on constructors only.

Reply:
Removed it.